src/aqua/chain/chain.aqua

```diff
@@ -1,8 +1,15 @@
 module Chain declares ChainInfo
```

Comment:
this file now basically duplicates rust definition. But it has nicer types! So I hesitate to remove it.

Reply:
why not type alias commensurately in chain_info.rs?

Reply:
because `#[marine]` doesn't support type aliases